Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml

```diff
@@ -33,6 +33,28 @@
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpskirt/secformalskirt.rsi
 
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpskirtQMFormal
+  name: logistics officer's formal dress
+  description: Inspired by the logistics officers of military's past, the perfect outfit for supplying a formal occasion.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/qmformaldress.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/qmformaldress.rsi
+
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpskirtHoPFormal
+  name: head of personnel's formal dress
+  description: A well-made dress with gold buttons and a red trim. Might as well look sharp when you blow up.
```

Comment:
can you change blow up to something like service disaster related

Reply:
Sure thing! Would you like me to change the description on the suit as well?